index.js

```diff
@@ -42,7 +42,7 @@ const wrapWord = (rows, word, columns) => {
 
 		if (ESCAPES.has(character)) {
 			isInsideEscape = true;
-			isInsideLinkEscape = characters.slice(index + 1).join('').startsWith(ANSI_ESCAPE_LINK);
+			isInsideLinkEscape = word.slice(index + 1).startsWith(ANSI_ESCAPE_LINK);
```

Comment:
`.slice()` operates on code units while the `[...word]` splits on codepoints. That's a subtle difference.

```
> '🌍'.slice(0, 2).length
< 2
> [...'🌍'].length
< 1
```

Reply:
Good catch, not too familiar with unicode/string particulars. 

Will add a test to this effect and update PR